kickoff/jsonexport.py

```diff
@@ -0,0 +1,244 @@
+from kickoff import app
+
+from flask import jsonify, render_template
+
+from kickoff.model import getReleases
+from kickoff.firefoxversions import firefoxversions
+
+from kickoff.thunderbirddetails import primary_builds as tb_primary_builds, beta_builds as tb_beta_builds
+from kickoff.thunderbirdversions import thunderbirdversions
+
+from kickoff.mobileversions import mobileversions
+
+import config
+
+
+def getFilteredReleases(product, categories, ESR_NEXT=False, lastRelease=None, withL10N=False):
```

Comment:
OK, I had a look and we can't use it because we have too many special cases. Some examples: 3.6.0build1, 3.0rc1 or 3.6.3plugin1 are not understood by this regexp.